wait-for.bats

```diff
@@ -79,7 +79,7 @@
 	sleep $((timeout + 1))
 	EOF
   chmod +x delay
-  nc -lk -p 80 -e $(pwd)/delay & ncpid=$!
```

Comment:
This probably breaks the tests when running them without the Docker container. I'm considering some kind of lookup or "fallback" mechanism. Maybe the test-env container should override the default `nc` binary with the `nmap-ncat`. What do you think?